.travis.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         - which cargo-kcov || cargo install cargo-kcov
         # We need to run cargo clean to avoid caching issues
         - cargo clean
-        - RUSTFLAGS="-C link-dead-code" cargo build --features=decode_test,decode_test_dav1d,quick_test --tests --verbose
+        - RUSTFLAGS="-C link-dead-code" cargo build --features=decode_test_dav1d,quick_test --tests --verbose
```

Comment:
Do we want to keep both enabled for the quick tests?

Reply:
Yes, I think I'll reduce this down to just splitting the ignored tests job.